api/endpoint-tests/states/get.js

```diff
@@ -75,7 +74,7 @@ tap.test('states/:id endpoint | GET /states/:id', async getStateTest => {
   await db().seed.run();
 
   unauthenticatedTest('get', url, getStateTest);
-  unauthorizedTest('get', url, getStateTest);
+  // unauthorizedTest('get', url, getStateTest);
```

Comment:
should we delete this?

Reply:
Yep.  Thought I'd gotten those.  Nice catch.